pages/Reviews/connector.js

```diff
@@ -14,6 +15,7 @@ import { getProductReviews } from '@shopgate/pwa-common-commerce/reviews/selecto
  * @return {Object} The extended component props.
  */
 const mapStateToProps = state => ({
+  rating: getProductRating(state),
```

Comment:
I'd say it should receive props from parent, since both Header and List is a general meant to be a general use components (that's why it's in the Components folder).

It's true that currently both of them receive current product data. But since they are general use they can be used in for example some special list like favourite list page or "your orders" page, where then product data would be passed. This way implementing those kind of pages would require little or no changes in those components.

But it's up to you, if you have strong opinion about it, I'm happy to move the connector job to components.